ivy/functional/ivy/layers.py

```diff
@@ -2340,3 +2340,75 @@ def _get_num_padded_values(i, p, n, k, s):
     return max(0, left_padding - current_index) + max(
         0, current_index + k - n - left_padding
     )
+
+
+# TODO add paddle backend implementation back,
+#  once paddle.argsort uses a stable algorithm
```

Comment:
let's probably link an issue on their github repository which we've created or which already existed addressing this so that we can add the `paddle` backend implementation once the issue is closed

Reply:
awesome, thanks!